JULIERAJ/KIDS-FIRST-Front
# Refactor/4/css styles and imports of components

## One Line Description
- Set up jsconfig to shorten path lengths.
- Include an index.js file for every component.
- Eliminate unused styles and convert them to module.css.

## Test Steps
1. Once PR please run `npm i `to install dependencies.
2. Open the application.
3. Check if you don't have any warnings such as eslint and if the paths of some components can't be resolved or are undefined.
